panic because of missing logger

using latest master (after #32) in pion/webrtc leads to a crash because the logger isn't setup correctly somehow: 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x15bc13e]

goroutine 90 [running]:
github.com/pion/quic.(*TransportBase).acceptStreams(0xc0002980a8)
	/Users/cryptix/go/pkg/mod/github.com/pion/quic@v0.1.2-0.20200714121404-405825cda397/transportbase.go:114 +0xee
created by github.com/pion/quic.(*TransportBase).startBase
	/Users/cryptix/go/pkg/mod/github.com/pion/quic@v0.1.2-0.20200714121404-405825cda397/transportbase.go:64 +0x53
exit status 2
```

https://github.com/pion/quic/blob/405825cda397c816d3e82536e138042b7beab0c9/transportbase.go#L114

--- a/transportbase.go
+++ b/transportbase.go
@@ -33,6 +33,12 @@ type Config struct {
 // should instead use the methods on quic.Transport or
 // webrtc.QUICTransport to setup a Quic connection.
 func (b *TransportBase) StartBase(conn net.Conn, config *Config) error {
+	lf := config.LoggerFactory
+	if lf == nil {
+		lf = logging.NewDefaultLoggerFactory()
+	}
+	b.log = lf.NewLogger("quic-wrapper")
+
 	cfg := config.clone()
 	cfg.SkipVerify = true // Using self signed certificates; WebRTC will check the fingerprint
 

